docs/index.md

```diff
@@ -98,7 +98,7 @@ or as a development dependency for your project:
 $ npm install --save-dev mocha
 ```
 
-> As of v7.0.0, Mocha requires Node.js v8.0.0 or newer.
+> As of v7.0.0, Mocha requires Node.js v10.0.0 or newer.
```

Comment:
It will be a `semver-major`: As of v8.0.0, ...

Reply:
Sure. I will update the PR asap :-)

Reply:
this is still incorrect.